subsys/net/lib/nrf_cloud/src/nrf_cloud_sanity.c

```diff
@@ -30,9 +30,3 @@
 #endif
 
 #endif /* CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES */
-
-
-/* Ensure correct data type size */
-
-BUILD_ASSERT_MSG(sizeof(enum nrf_cloud_sensor) == sizeof(u8_t),
```

Comment:
cc @lemrey

Reply:
When I started working to port this code I was unfamiliar with it so when changing this type to an `enum` I added an assertion to make sure the size was the same, in case it mattered. It didn't matter, so it can be removed now :)